app/templates/components/nav-bar.hbs

```diff
@@ -48,7 +48,7 @@
       <i class="large content icon"></i>
     </a>
     {{#link-to 'index' class='item' activeClass=''}}
-      <div class="ui header small text">{{config.appName}}</div>
```

Comment:
Use config please. Settings is asynchronous so it'll take time to show up

Reply:
We are using settings for various importing the name and tagline for the appName and various other input we need.
Also setting is globally injected into app as services, So it should not make much of a difference.

Reply:
`Also setting is globally injected into app as services, So it should not make much of a difference.`

I don't know any of that. I know that an AJAX request is made for that. This thinking has lead to how slow the app is.

And if settings are meant to be used here, why is there an option of config.appName?